py/kubeflow/testing/cloudprovider/aws/prow_artifacts.py

```diff
@@ -278,94 +270,3 @@ def finalize_prow_job_to_s3(bucket, workflow_success, workflow_phase, ui_urls):
 
   return test_success
 
-
-def main(unparsed_args=None):  # pylint: disable=too-many-locals
```

Comment:
removing unused `main()` leaves functions `copy_artifacts_to_s3()` and `create_pr_symlink_s3()` dead, which can also be removed

Reply:
So from the python file, all I want is function import and use in the test cases

E.g, 

https://github.com/kubeflow/kfctl/blob/485a7c034e0a198d3ef1e8c6f441b4ac37b90013/py/kubeflow/kfctl/testing/ci/kfctl_e2e_workflow.py#L488

Reply:
Yeah I noticed we actually use those main() function in e2e test, e.g, https://github.com/kubeflow/kfctl/blob/485a7c034e0a198d3ef1e8c6f441b4ac37b90013/py/kubeflow/kfctl/testing/ci/kfctl_e2e_workflow.py#L488

Let me revert it back